### ISSUE TITLE
crawling timestamps to reflect actual test time

### ISSUE THREAD

**Issue by matti** [sitespeedio/sitespeed.io#2766]
```
--browsertime.iterations=1
--crawler.depth=2
--crawler.maxPages=10
```

all 10 html report pages say that `Tested 2019-11-07 17:29:24` (when the testing started, not when the page was actually tested) - would it be possible to have the timestamp to reflect the actual time, since it varies a lot when pages are tested sequentially?

**Comment by soulgalore** [comment]
Yes I agree. I think each individual run page (clicking on the number) has the right timestamp?

**Comment by matti** [comment]
There are no other numbers than 1 (I have set iterations to be 1) - this makes it even more confusing, when 10 crawled pages have the same time and it took atleast 50s to crawl them.

But yes, with multiple iterations there would be different times, but again not for different crawled pages.

**Comment by soulgalore** [comment]
I've checked the code we actually to get the timestamp for the summary but doesn't do that correct so it fallbacks to the default timestamp.

**Comment by soulgalore** [comment]
This was fixed in 11.1.0!
